gotthard/cli.py

```diff
@@ -245,7 +243,7 @@ def setup_tunnel(user, odd_host, remote_host, remote_port, tunnel_port):
                    '{}@{}'.format(user, odd_host),
                    '-N']
 
-    process = subprocess.Popen(ssh_command, preexec_fn = os.setpgrp)
+    process = subprocess.Popen(ssh_command, preexec_fn=os.setpgrp)
```

Comment:
PEP8: Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.